manifests/repo/el.pp

```diff
@@ -26,4 +28,11 @@
       enabled  => 1,
     }
   }
+
+  if $yum_proxy {
+    Yumrepo['jenkins'] {
+      proxy => $yum_proxy,
+    }
```

Comment:
`repo_proxy` might be a bit better long term since fedora 22 has replaced yum with dnf.